src/Abp.OnlineClientStore.Redis/RealTime/Redis/DefaultRedisOnlineClientStoreSerializer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Abp.Dependency;
+using Newtonsoft.Json;
+using StackExchange.Redis;
+
+namespace Abp.RealTime.Redis
+{
+    /// <summary>
+    ///     Default implementation uses JSON as the underlying persistence mechanism.
+    /// </summary>
+    public class DefaultRedisOnlineClientStoreSerializer : IRedisOnlineClientStoreSerializer, ITransientDependency
```

Comment:
what is the different between this and `IRedisCacheSerializer`?

Reply:
DefaultRedisCacheSerializer uses AbpCacheData as wrapper